### DISCUSSION TITLE
Neo4j

### DISCUSSION THREAD

**Discussion by AruneshSingh** [superlinked/VectorHub#96] [VDB Comparison]
Hey everyone, this is the start of discussions related to [Neo4j](https://neo4j.com/) and their attributes as compared in the table [here](https://vdbs.superlinked.com). 

To add to the discussion please check if there's already a comment with the attribute you want to discuss about, and reply in the thread. Otherwise please create a new comment for the attribute you want to discuss about.

To create an issue, please have a look at the [README](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/README.md) for contributing guidelines.

Let's all try to have a fruitful discussion!

**Comment by Ratgor** [comment]
I have read on Neo4j site about it has open-source (community edition) and free-of-charge, is it true? currently in table it is marked as not 
(or we may assume several lines if free-or-not  versions have significant diffs in features)
